canonical/lxd-ui
# Fix long item names

## Done

- truncate instance name in the list view to avoid breaking into more than 1 line
- break instance name into multiple lines in modals and notifications
- change all notifications to print the item name in bold letters or as link
- adjusted instances, storage and snapshots. We don't use the `InstanceName` component for queued notifications, those can be text only and not include other react components.

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described in the [Readme](https://github.com/canonical/lxd-ui#setting-up-for-development).
2. Perform the following QA steps:
    - check the instance name in the instance list with long names
    - check the modals and notifications on start/stop/reboot/pause/delete of instance
